app/views/comments/_comment.html.slim

```diff
@@ -0,0 +1,33 @@
+.post__main__comments__list__detail__wrapper.container.mb-4 id="comment-#{comment.id}"
+  .row.gx-3
+    .align-top.col-1
+      i class="fa-solid fa-user fa-lg border border-secondary text-secondary p-2 rounded-5 float-end"
+    .col-11
+      .post__main__comments__list__detail.border.p-2.rounded.w-100
+        .mb-2.post__main__comments__list__detail__infomation
+          span
+            = comment.user.username
+            |  •
+            small.ms-2.text-secondary
+              = comment.created_at.to_date.to_s(:long)
+
+          - if current_user && (current_user == comment.user || current_user == @post.user || current_user.admin?)
```

Comment:
If condition logics is complex and long, you can create a helper method (in this model is CommentHelper) and use it in view, create it own helper method will help easier to understand what the condition do